queries/css/highlights.scm

```diff
@@ -19,6 +19,8 @@
  (universal_selector)
  ] @type
 
+((class_selector) @type (#set! "priority" 110))
```

Comment:
Does it not work without the priority?

Reply:
can we do it for `class_name` and `id_name` instead?

Remove `class_name` and `id_name` from

```
[
 (class_name)
 (id_name)
 (property_name)
 (feature_name)
 (attribute_name)
 ] @property
```

and add

```
[
 (class_name)
 (id_name)
] @type
```

Reply:
While we're at it. Maybe also do:

```
(class_selector "." @punctuation.special)
(id_selector "#" @punctuation.special)
```

Reply:
id isn't really type tho, more of a specifier/variable

Reply:
```css
#id-name,
.class-name {
}
```

Yeah, they're not really `@type`... same for `class_name`.

But I'm not sure if any of the existing captures fits them.

They're not `@variable` either, css has variable and they're a different thing.

They are specifier. But is there a capture group for that?

Reply:
maybe `@attribute`, idk css

Reply:
attribute too are different thing 😂 (updated the example in previous comment)